src/main/java/dbaccessor/message/AttachmentAccessor.java

```diff
@@ -30,4 +30,23 @@ public boolean checkIfAttachmentIdExists(long attachmentId) {
  
         return !resultSet.isEmpty();
     }
+
+    /**
+     * Returns details of Attachment with the given AttachmentId.
+     * Details include:
+     * <ol>
+     * <li> AttachmentId </li>
+     * <li> FileName </li>
+     * <li> FileType </li>
+     * <li> FileSize </li>
+     * </ol>
+     */
+    public Attachment getAttachment(long attachmentId) {
+
+        String SQLStatment = "SELECT * FROM Attachment WHERE AttachmentID=@attachmentId";
+        Statement statement = Statement.newBuilder(SQLStatment).bind("attachmentId").to(attachmentId).build();
+        List<Attachment> resultSet = spannerTemplate.query(Attachment.class, statement, null);
+ 
+        return resultSet.get(0);
```

Comment:
What if the given id is invalid and no attachment is found. This will throw an exception.

Reply:
All `DBAcessors` are written with the assumption that the passed parameters are valid. This is because all of them are checked before-hand in the `Controllers` to return `custom` error responses. 

In this case, the `attachmentId` is passed directly after retrieving it from the `Message` object from the `Spanner` DB. Hence, it wasn't checked in the `Controller`.

Reply:
Sure thing.